cmake/ImportLLVM.cmake

```diff
@@ -105,3 +113,6 @@ if(CMAKE_SIZEOF_VOID_P EQUAL 4 OR CA_BUILD_32_BITS OR
   add_definitions(-D_LARGEFILE_SOURCE)
   add_definitions(-D_FILE_OFFSET_BITS=64)
 endif()
+
+# Set LLVM Include directories for all targets
+include_directories(SYSTEM ${LLVM_INCLUDE_DIRS})
```

Comment:
Does this mean we also get `LLVM_INCLUDE_DIRS` added for targets that previously did not have it? That's probably fine, but good to make sure.

Reply:
When you say targets I assume you mean cmake targets. If so, then yes it does - although we wouldn't link with it and it would fail then.

Reply:
It would not necessarily fail: I've tried to use header-only LLVM functions before in places that don't and shouldn't have a dependency on LLVM. It is useful, though not absolutely required, for that to fail as soon as possible.

Reply:
Fair enough in theory you could include some llvm, but you'd need to do it pretty knowingly via include <llvm/....>